lib/player.js

```diff
@@ -35,6 +35,17 @@ Player.prototype = {
     }
   },
 
+  eatPlayer: function(players){
+    for(var i = 0; i < players.length; i++) {
+      var xDiff = this.x - players[i].x;
+      var yDiff = this.y - players[i].y;
+      var distance = Math.sqrt( xDiff*xDiff + yDiff*yDiff);
+      if((distance < this.mass * 0.8) && (players[i] !== this) && (this.mass * 0.8 > players[i].mass)) {
```

Comment:
nitpicky, but can we organize this line so it's has a line break after each of the &&s @Jpease1020 ?